lang/LangSource/SC_TerminalClient.cpp

```diff
@@ -124,7 +124,7 @@ void SC_TerminalClient::printUsage()
 			"   -s                             Call Main.stop on shutdown\n"
 			"   -u <network-port-number>       Set UDP listening port (default %d)\n"
 			"   -i <ide-name>                  Specify IDE name (for enabling IDE-specific class code, default \"%s\")\n"
-			"   -a                             Standalone mode\n",
+			"   -a                             Standalone mode (exclude SCClassLibrary and user and system Extensions folders from search path)\n",
```

Comment:
@miguel-negrao being picky here but why not `class library` and `extensions` instead of capitalized ones?